src/java/Test.java

```diff
@@ -155,6 +157,25 @@ public static void testRemoveDuplicatesUnsortedList() {
         System.out.println("---------------------------------------------");
         singlyLinkedList.removeAll();
     }
+
+    public static void testBinarySearchAlgorithms() throws Exception {
```

Comment:
Create two test methods, one for isSorted, one for binarySearch method. Don't create tests for class